src/js/video.js

```diff
@@ -208,6 +221,29 @@ class Video {
 		this.containerEl.appendChild(this.placeholderEl);
 	}
 
+	updatePlaceholder() {
+		this.placeholderImageEl.src = this.posterImage;
+
+		if (this.placeholderTitleEl) {
+			this.placeholderTitleEl.textContent = this.videoData && this.videoData.name;
+		}
+	}
+
+	update(newOpts) {
+		this.videoEl && this.videoEl.pause();
+		this.clearCurrentlyPlaying();
+
+		this.opts = Object.assign(this.opts, { data: null }, newOpts);
+
+		return this.getData().then(() => {
+			if (this.placeholderEl) {
+				this.updatePlaceholder();
```

Comment:
The video element will only be added after clicking the placeholder, so there's no video to update at this time.